.flake8

```diff
@@ -1,5 +1,6 @@
 [flake8]
 #E266 too many leading '#' for block comment
 #E722 do not use bare 'except'
-ignore = E266,E722
+#W503 conflcits with updated rule W504

```

Comment:
I'd break this out into a separate PR if really needed

Reply:
this change is fine as it is also in pytool-library.  
I would suggest we align the comment to match.
It looks like this:   
#W503 line break before binary operator, see https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

see https://github.com/tianocore/edk2-pytool-library/blob/master/.flake8